cabal.project

```diff
@@ -1 +1,2 @@
 packages: garn.cabal
+tests: True
```

Comment:
Does this affect production builds? If yes, then you may not want this change.

Reply:
Our production builds are done with `nix` and this doesn't seem to affect that. What's the workflow you're using where this is beneficial?

Reply:
This ensures, that a `cabal build` always builds all test suites, without having to do a `cabal configure --enable-tests` first.  It also protects you against https://github.com/haskell/cabal/issues/5079, https://github.com/haskell/cabal/issues/7883 (yes, that's true, `cabal test all` does not necessarily build all your tests, https://github.com/haskell/cabal/issues/5079#issuecomment-751373847).

With nix, I am not sure how essential this still is for my workflow. So far I used a combination of `cabal`-targets, `just`-targets and `nix profile install` (which still fails for me, but populates the nix store, which I then can reference manually).

Not sure what exactly `nix profile install` does, but given that the `justfile` runs `cabal`-targets, I would probably tend to `tests: true` so that `cabal` always solves for a consistent set of dependencies, not sure.

Reply:
Sounds good. When I'm working on `garn`, `nix` provides all dependencies, so hopefully the cabal solver just finds all the necessary dependencies already installed and just uses them. But being explicit about that we *always* want it to solve for the test-suite seems good.